.github/workflows/sdk-resources-connections-connections.yml

```diff
@@ -67,6 +67,11 @@ jobs:
           [ -f "../../.azureml/config" ] && cat "../../.azureml/config";
           papermill -k python connections.ipynb connections.output.ipynb
       working-directory: sdk/python/resources/connections
```

Comment:
Please do not edit the workflows file manually, this change will be overwritten on next auto generation of the workflows.

Reply:
didn't manually edit, the changes are in readme.py as well